tests/test_watchmaker.py

```diff
@@ -8,6 +8,11 @@
 import pytest
 import yaml
 
+try:
+    from unittest.mock import patch
```

Comment:
Can you put a comment on why the try/except is necessary? I assume it's a python2/3 thing?